### ISSUE TITLE
"Reboot" button in setup after a repair does nothing

### ISSUE THREAD

**Issue by UweKeim** [dotnet/aspnetcore#36857]
### Description

Having used the "Repair" option of Hosting Bundle 3.1.19 and Hosting Bundle 5.0.10 they both showed a "Reboot" button on _some_ systems.

(Context was [this issue](https://github.com/dotnet/runtime/issues/1868#issuecomment-920792984))

Whenever I clicked the "Reboot" button, exactly nothing happend. Neither did the system reboot, nor was an error message shown.

This is a screenshot of on a German system ("Neu starten" is the reboot button).

![image](https://user-images.githubusercontent.com/415439/133604089-7e01d9a0-3d89-465b-8c27-bb2943a17bab.png)

In the end, I did a manual reboot.

### Configuration

OS: Windows Server 2019 German

### Expectation

I did expect that when clicking the "Reboot" button, the system actually reboots.

### Update 1

On some systems, the "Reboot" button actually works correctly. So this doesn't seem to be a general issue.

**Comment by adityamandaleeka** [comment]
@wtgodbe Can you please investigate?

**Comment by wtgodbe** [comment]
@UweKeim I've been unable to reproduce this, would you be able to upload setup logs from the repair of 3.1.19? They should be in `C:\Users\<username>\AppData\Local\Temp`

**Comment by UweKeim** [comment]
I've _tried_ to recognize the server where this happened, but am not sure whether I found the right one. I've checked several once and only found one with actual log files in `C:\Users\<username>\AppData\Local\Temp`.

I've attached them here:

> [dotnet5.zip](https://github.com/dotnet/aspnetcore/files/7272788/dotnet5.zip)

Looking through them it seems that everything worked correctly, so I'm really unsure whether these are the erroneous log files.
